backend/apps/sandbox/views.py

```
@@ -235,11 +310,102 @@ def get_queryset(self):
+        })
+
+
+# ============================================================
+# EXECUTION TRACKER ADMIN (For testing)
+# ============================================================
+
+class ClearExecutionView(APIView):
+    """
+    Clear execution from cache (for testing/admin).
+    """
+    permission_classes = [permissions.IsAuthenticated]
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)